src/components/Contacts/ContactDetails/ContactDetailsTab/Mailing/AddAddressModal/AddAddressModal.tsx

```diff
@@ -114,7 +114,11 @@ export const AddAddressModal: React.FC<EditContactAddressModalProps> = ({
                   ...dataFromCache.contact.addresses,
                   nodes: [
                     ...dataFromCache.contact.addresses.nodes,
-                    { ...createdAddressData?.createAddress?.address },
+                    {
```

Comment:
Instead of adding these lines, can we add the necessary fields to the `CreateContactAddress` mutation. That will load them from the server after creating the address.